### PR TITLE
Reorganize code for improved understandability

### PR DESCRIPTION
**Pull request by bdarfler**

### CODE REVIEW

**Comment by bdarfler** [comment]
@joshcough @jsolmon no logic changes here, just tried to clean up the code a bit based on some better patterns I saw out there (mostly rtimush/sbt-updates)

**Comment by jsolmon** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -72,5 +72,5 @@ dynamoDBLocalSharedDB := true
 The default on stop is to cleanup any data directory if specified. This can be changed using
 
 ```
-cleanDynamoDBLocalAfterStop := false
+dynamoDBLocalCleanAfterStop := false
```
Comment (joshcough) [inline README.md]: +1

**Comment by joshcough** [comment]
Very nice!

**Pull request merged**